src/Aspire.Hosting.Docker/Aspire.Hosting.Docker.csproj

```diff
@@ -7,6 +7,7 @@
     <PackageTags>aspire hosting kubernetes</PackageTags>
     <Description>Kubernetes publishing for .NET Aspire.</Description>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <NoWarn>$(NoWarn);CS8002</NoWarn>
```

Comment:
What's this?

Reply:
Can't use k8s.models without it - as they still don't sign the KubernetesClient library
```bash
0>CSC: Error CS8002 : Referenced assembly 'KubernetesClient, Version=16.0.0.0, Culture=neutral, PublicKeyToken=null' does not have a strong name.
0>------- Finished building project: Aspire.Hosting.Docker. Succeeded: False. Errors: 1. Warnings: 0
```